src/js/utils.js

```diff
@@ -0,0 +1,48 @@
+import { LOTTO } from "./constants.js";
+
+export const getLottoRandomNumber = () => {
+  return Math.floor(Math.random() * (LOTTO.MAX_NUMBER - LOTTO.MIN_NUMBER) + 1);
+};
+
+export const calculateEarningRates = (earnings, investments) =>
+  Math.trunc(((earnings - investments) / investments) * 100);
+
+const getCorrectCount = (lottoNumbers, winningNumbers) => {
+  let count = 0;
+  lottoNumbers.forEach((number) => {
+    if (winningNumbers.includes(number)) {
+      count++;
+    }
+  });
+
+  return count;
+};
```

Comment:
```suggestion
const getCorrectCount = (lottoNumbers, winningNumbers) => {
  return  lottoNumbers.map((number) => winningNumbers.includes(number)).filter(Boolean).length
};
```

좀 더 함수형스럽게 표현해볼 수 있을 것 같아요